.gitignore

```diff
@@ -4,3 +4,4 @@
 /src/framework/preferences/css/*.css
 /.bundle/
 /.vagrant/
+report.tap
```

Comment:
We talked about this in the [fluid-work irc channel](https://botbot.me/freenode/fluid-work/2016-08-23/?msg=71810698&page=2). I believe you'll need to update the Grunt clean task to also cleanup the report.tap file.